src/lib/viewers/image/MultiImageViewer.js

```diff
@@ -148,7 +148,14 @@ class MultiImageViewer extends ImageBaseViewer {
         this.singleImageEls[index].setAttribute('data-page-number', index + 1);
         this.singleImageEls[index].classList.add(CLASS_MULTI_IMAGE_PAGE);
 
-        this.singleImageEls[index].src = imageUrl;
+        this.downloadUrl = imageUrl;
+
+        fetchRepresentationAsBlob(imageUrl)
+            .then((blob) => {
+                const srcUrl = URL.createObjectURL(blob);
+                this.singleImageEls[index].src = srcUrl;
```

Comment:
Now that we're using axios, we can optimize here in the future by cancelling failed requests after we detect an initial failure.